applications/StructuralMechanicsApplication/custom_python/add_custom_response_functions_to_python.cpp

```diff
@@ -68,13 +69,9 @@ void  AddCustomResponseFunctionUtilitiesToPython(pybind11::module& m)
         .def(py::init<ModelPart&>());
 
     // Response Functions
-    py::class_<AdjointStructuralResponseFunction, AdjointStructuralResponseFunction::Pointer>
+    py::class_<AdjointStructuralResponseFunction, AdjointStructuralResponseFunction::Pointer, AdjointResponseFunction>
```

Comment:
I think you can remove this from here completely, because we dont need it in python

Reply:
That is right.

Reply:
@armingeiser when removing it I am not sure how to modify the stress, displacement and strain-energy response. When I remove it compiling works but if I run a python file I always get errors concerning the base type which is for these classes the ```AdjointStructuralResponseFunction```.

Reply:
You need to replace the `AdjointStructuralResponseFunction` in the py::class_ definition with the base class from the core.